Godeps

```diff
@@ -3,7 +3,6 @@ github.com/BurntSushi/toml 99064174e013895bbd9b025c31100bd1d9b590ca
 github.com/bmizerany/pat c068ca2f0aacee5ac3681d68e4d0a003b7d1fd2c
 github.com/boltdb/bolt 4b1ebc1869ad66568b313d0dc410e2be72670dda
 github.com/cespare/xxhash 4a94f899c20bc44d4f5f807cb14529e72aca99d6
-github.com/clarkduvall/hyperloglog 2d38f733946d0a1f2e810513c71b834cbeba1480
```

Comment:
I couldn't figure out where this came from. `statik` doesn't seem to use it, but gdm also didn't pick it up.